MiniGames/MiniGames/MiniGames.cpp

```diff
@@ -0,0 +1,140 @@
+// MiniGames.cpp -- game rock/paper/scissors (with computer)
+
+#include <ctime>
+#include <iostream>
+#include <iomanip>
+
+
+#include "round_result.cpp"
+#include "choice_generation.cpp"
+
+
+int main()
+{
+	std::srand(std::time(nullptr));
+
+	const int END_PROGRAM = -1;
+	const unsigned int NUMBER_SPACES = 10;
+	const char ROCK_SYMBOL = 'r', PAPER_SYMBOL = 'p', SCISSORS_SYMBOL = 's';
+	const char DRAW_SYMBOL = 'd', WIN_SYMBOL = 'w', LOSS_SYMBOL = 'l';
+
+	
+	while (true)
+	{
+
+		int draws = 0, wins = 0, losses = 0;
+		int number_rounds = END_PROGRAM;
+
+
+		std::cout << "Enter" << " " << END_PROGRAM << " " 
+			<< "to exit or the number of rounds you want to play : " 
+			<< " ";
+		std::cin >> number_rounds;
+		std::cout << std::endl;
+
+		if (number_rounds == END_PROGRAM)
+		{
+			std::cout << "You entered" << " " << number_rounds
+				<< "," << " " << "the program closes." << std::endl;
+			std::cout << "GG!" << std::endl;
+			break;
+		}
+
+		if (number_rounds <= 0)
+		{
+			std::cout << "This number of rounds cannot exist, please enter a valid number." << std::endl << std::endl;
+			continue;
+		}
+
+		char user_choice = '0';
+		int pc_choice = -1;
```

Comment:
I know why, but here it looks strange that user_choice and pc_choice have different types)

Reply:
Simple conversion later would do the trick